lib/active_model/serializer.rb

```diff
@@ -6,6 +6,7 @@ class Serializer
     autoload :Configuration
     autoload :ArraySerializer
     autoload :Adapter
+    autoload :Lint
```

Comment:
Only because it's cheap.  I'd be perfectly happy to change it to a `require 'active_model/serializer/lint'`